dashboard/test/factories/factories.rb

```diff
@@ -544,18 +544,6 @@
       end
     end
 
-    trait :with_lti_authentication_option do
```

Comment:
Removing this trait, moving to factory below. This allows us to build the user's authentication_options before creating the user, rather than first creating a user, then creating authentication_options. This mimics how we are creating LTI users.